src/pages/RealmScans/main.ts

```diff
@@ -0,0 +1,76 @@
+import { pageInterface } from '../pageInterface';
+
+export const RealmScans: pageInterface = {
+  name: 'RealmScans',
+  domain: 'https://realmscans.com',
+  languages: ['English'],
+  type: 'manga',
+  isSyncPage(url) {
+    return j.$('#screenmode').length > 0;
+  },
+  isOverviewPage(url) {
+    return utils.urlPart(url, 3) === 'series';
+  },
+  getImage() {
+    return j.$('div.thumb > img').attr('src');
+  },
+  sync: {
+    getTitle(url) {
+      return j.$('ol > li:nth-child(2) > a > span').text();
+    },
+    getIdentifier(url) {
+      const temp_ident = j.$('ol > li:nth-child(2) > a').attr('href') || '';
+      return utils.urlPart(temp_ident, 4);
+    },
+    getOverviewUrl(url) {
+      return j.$('ol > li:nth-child(2) > a').attr('href') || '';
+    },
+    getEpisode(url) {
+      const episodePart = utils.urlPart(url, 3);
+
+      const temp = episodePart.match(/-chapter-\d+/gim);
```

Comment:
`const temp = episodePart.match(/-chapter-(\d+)/im);`
if you drop the g in the regex then you can use groups and the element with the index [1] already contains number.

and if it does not match anything the array is null

`if(!temp) return NaN;`
`return Number(temp[1])`